nixos/modules/services/x11/xserver.nix

```diff
@@ -409,6 +409,16 @@ in
           if possible.
         '';
       };
+
+      enableCtrlAltBackspace = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to enable the DontZap option, which binds Ctrl+Alt+Backspace
```

Comment:
Seems like an implementation detail. As a user I didn't know that this was referred to as "zapping" in X, so I feel like people might miss this option. That said, you'd only change it from the default if you knew what it was, so maybe a name like that is ok...